crm_compassion/models/interaction_resume.py

```diff
@@ -181,8 +190,15 @@ def populate_resume(self, partner_id):
                             WHEN mail.bounced IS NOT NULL THEN 'bounced'
                         ELSE 'sent'
                         END tracking_status,
-                        mm.id as mass_mailing_id
+                        mm.id as mass_mailing_id,
+                        false as has_attachment
                         FROM "mail_mail_statistics" as mail
+                        FULL OUTER JOIN (
+                            SELECT m.id AS id, Count(ma.message_id) AS nb_attachment
+                            FROM mail_message AS m
+                            RIGHT JOIN message_attachment_rel ma ON ma.message_id = m.id
+                            GROUP BY m.id
+                        ) AS attachment ON mail.mail_mail_id = attachment.id
```

Comment:
Here you have "false as has_attachment" but you add a Full outer join, is it necessary?

Reply:
Forgot to add back before committing when testing